bench/utils/__init__.py

```diff
@@ -429,7 +429,7 @@ def find_org(org_repo):
 
 	for org in ["frappe", "erpnext"]:
 		res = requests.head(f"https://api.github.com/repos/{org}/{org_repo}")
-		if res.status_code == 400:
+		if res.status_code in (400, 403):
 			res = requests.head(f"https://github.com/{org}/{org_repo}")
```

Comment:
I think if the API returns 403 then the open route won't work either 🤔 
There could be a separate exception for it for the time being

Reply:
Nope. For the other URL, it is working fine.

<img width="685" alt="Screenshot 2022-04-08 at 14 28 55" src="https://user-images.githubusercontent.com/4463796/162406232-c32cc386-a9d2-4978-a6d1-997761f48e7c.png">

Only for API it is returning 403.